docs/fern/docs/pages/others/self-host.mdx

```diff
@@ -26,6 +26,26 @@ On a high level, Stack Auth is composed of the following services:
 - **Svix**: Used to send webhooks. Svix is open-source and can be self-hosted, but also offers a cloud hosted solution. More on Svix [here](https://svix.com)
 - **Email server**: We use [Inbucket](https://inbucket.org) as a local email server for development and a separate SMTP server for production. Any email service supporting SMTP will work.
 
+## Run with Docker
+
+Stack Auth provides a pre-configured Dockerfile that bundles both the dashboard and API backend into a single container. To complete the setup, you'll need to provide your own PostgreSQL database, and optionally configure an email server and Svix instance for webhook functionality.
+
+Here is an example of how to run the services with Docker:
+
+Get the example environment file [here](https://github.com/stack-auth/stack/tree/dev/docker/server/.env.example). See the full template [here](https://github.com/stack-auth/stack/blob/dev/docker/server/.env).
+
+```sh
+# Start a Postgres database
+docker run -d --name db -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=password -e POSTGRES_DB=stackframe -p 5432:5432 postgres:latest
+
+# Run the Docker container
+docker run --env-file your-env-file.env -p 8101:8101 -p 8102:8102 stackauth/server:main
+```
+
+Now you can open the dashboard at [http://localhost:8101](http://localhost:8101) and the API backend on port 8102.
+
+Now, login with your admin account on the dashboard and follow the [normal setup process](/getting-started/setup). Add `NEXT_PUBLIC_STACK_URL=https://your-backend-url.com` to your app's environment variables so that it connects to your API backend instead of the default Stack Auth API backend (https://api.stack-auth.com).
+
```

Comment:
Since we're building & publishing the Docker container to Dockerhub, can't we just tell people here to download that image? I think most people don't want to build the image from scratch

Reply:
This is the docker from docker hub, there is no building process